System.IO.Abstractions.TestingHelpers.Tests/MockFileSystemTests.cs

```diff
@@ -277,5 +277,20 @@ public void MockFileSystem_AddDirectory_TrailingSlashAllowedButNotRequired(strin
 
             Assert.IsTrue(fileSystem.FileExists(path2));
         }
+
+        [Test]
+        public void MockFileSystem_GetFiles_ThrowsCorrectExceptionForInvalidCharacters()
+        {
+            // Arrange
+            const string path = @"c:\";
+            var fileSystem = new MockFileSystem();
+            fileSystem.AddDirectory(XFS.Path(path));
+            
+            // Act
+            TestDelegate wrapped = () => fileSystem.Directory.GetFiles($"{path}{'\0'}.txt");
```

Comment:
I don't like the name `wrapped` - maybe something like `getFilesWithInvalidCharacterInPath`?

Reply:
Changed